packages/wallets/src/features/cfd/flows/CTrader/AddedCTraderAccountsList/AddedCTraderAccountsList.tsx

```diff
@@ -25,12 +39,18 @@ const AddedCTraderAccountsList: React.FC = () => {
                 >
                     <TradingAccountCard.Icon>{PlatformDetails.ctrader.icon}</TradingAccountCard.Icon>
                     <TradingAccountCard.Content>
+                        <LabelPairedChevronRightCaptionRegularIcon
+                            data-testid='dt_wallets_trading_account_chevron_icon'
+                            width={16}
+                        />
                         <Text align='start' size='sm'>
                             {PlatformDetails.ctrader.title}
                         </Text>
-                        <Text align='start' size='sm' weight='bold'>
-                            {account?.display_balance}
-                        </Text>
+                        {totalBalance !== undefined && (
+                            <Text align='start' size='sm' weight='bold'>
+                                {displayBalance}
+                            </Text>
+                        )}
```

Comment:
@hasan-deriv could we pls use WalletMoney component to show balance, we are trying to use it in wallets now :pray: